README.md

```diff
@@ -47,20 +56,85 @@ and ask!
 
 ## When to Create a New Enhancement
 
+Enhancements should be related to work to be implemented in the near
+future. If you have an idea, but aren't planning to implement it right
+away, the conversation should start somewhere else like the aos-devel
+mailing list.
+
 Create an enhancement here once you:
 
 - have circulated your idea to see if there is interest
 - (optionally) have done a prototype in your own fork
 - have identified people who agree to work on and maintain the enhancement
   - many enhancements will take several releases to complete
 
-## Why are Enhancements Tracked
-
-As the project evolves, its important that the OKD community understands how we
-build, test, and document our work.  Individually it is hard to understand how
-all parts of the system interact, but as a community we can lean on each other
-to build the right design and approach before getting too deep into an
-implementation.
+## How are Enhancements Reviewed and Approved?
+
+The author of an enhancement is responsible for managing it through
+the review process, including soliciting feedback on the pull request
+and in meetings, if necessary.
+
+Each enhancement should have at least one "approver" and several
+reviewers designated in the header of the document.
+
+The approver assists authors who may not be familiar with the process,
+the project, or the maintainers. They may provide advice about who
+should review a specific proposal and point out deadlines or other
+time-based criteria for completing work. The approver is responsible
+for recognizing when consensus has been reached so that a proposal is
+ready to be approved, or formally rejected. In cases where consensus
```

Comment:
i know we didn't come up with a great mechanism, but i'd still like to see some proposed system for tracking when a reviewer is formally saying "i'm ok with this" so that an approver understands that.

Reply:
I'll give that more thought and add those details in a separate PR. That'll give us a clear space to discuss the process changes, vs. this PR that's trying to document the current expectations.